package/MDAnalysis/guesser/DefaultGuesser.py

```diff
@@ -0,0 +1,155 @@
+from .base import GuesserBase
```

Comment:
Please add the standard comment header to the file (copy it verbatim from eg universe.py, don’t worry about changing any years)

Can please you change the file name to sometimes lower case? We don’t use camel case for file names.

Reply:
Sorry, I wasn't very clear on the file naming: I meant to change it from DefaultGuesser.py to default_guesser.py.

However, if there's a specific reason why it should looke like defaultGuesser.py then leave it — @jbarnoud ?

Reply:
Agreed with @orbeckst. This is a matter of naming conventions at the language level, conventions that are formalised in the [PEP8 document](https://peps.python.org/pep-0008/). PEP8 describes originally the conventions that developers of the python language should follow when editing the standard library. The conventions are good, though, so they became the de facto conventions used for basically anything in python.

When it comes to naming conventions, PEP8 says that:
* variables and modules are written as `variable_name`
* constants are written as `CONSTANT_NAME` and should be defined at the beginning of the file under the imports
* class names are written as `ClassName`.